Tables20000.m

```diff
@@ -0,0 +1,20 @@
+%FIRST RUN CrossCheckStata in order to have the results
```

Comment:
This is ok for now. I will ask you later to move all the testing to our Dropbox folder. If we keep some testing/examples in the repos for publication, it may be a good idea to put them in a subfolder. This is just a heads up for what's coming next so you keep it in mind while developing the code.

Reply:
Perfect I will now star that new folder to have the examples of the efficiency of the code.